Dockerfile

```diff
@@ -55,6 +57,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone \
 	imagemagick \
 	poppler-utils \
 	ghostscript \
+	wget \
```

Comment:
Not required for the application. If only used during mage build, you may delete it at the end to keep the image small

Reply:
removed it as it wasn't used anywhere